File: server/apps/api-service/src/main/java/com/sharetimer/apiservice/domain/model/Timestamp.java
package com.sharetimer.apiservice.domain.model;

import java.time.Instant;

import lombok.Builder;
import lombok.Getter;

/**
 * Timestamp domain model
 */
@Getter
@Builder
public class Timestamp {

  private Long id;
  private Instant targetTime;
  private Instant capturedAt;
  private Instant createdAt;
  private Instant updatedAt;

}

File: server/apps/api-service/src/main/kotlin/com/sharetimer/apiservice/domain/model/Timestamp.kt
package com.sharetimer.apiservice.domain.model

import java.time.Instant

/**
 * Timestamp domain model
 */
data class Timestamp(
    val id: Long? = null,
    val targetTime: Instant,
    val capturedAt: Instant,
    val createdAt: Instant? = null,
    val updatedAt: Instant? = null,
)
